internal/messages/errormessages.go

```diff
@@ -61,4 +61,5 @@ const (
 	Unit               = "unit"
 	Max                = "max"
 	Timeshift          = "timeshift"
+	TrustIDGroupName   = "TrustIDGroupName"
```

Comment:
This parameter is the only one to start with an uppercase letter?

Reply:
fixed